inspirehep/modules/workflows/workflows/manual_merge.py

```diff
@@ -80,7 +79,7 @@ def start_merger(head_id, update_id, current_user_id=None):
     workflow_object.extra_data.update(data)
 
     # preparing identifiers in order to do less requests possible later
-    head_source = get_head_source(head.id) or merger_get_source(head)
+    head_source = get_head_source(head)
```

Comment:
I'd suggest to completely remove the ``get_head_source`` from manual merge

Reply:
also remove it from https://github.com/inspirehep/inspire-next/blob/4a123080ac1d2cdd6f445a3104f8ef06b99edb44/inspirehep/modules/workflows/tasks/manual_merging.py#L65